Remove tinc-static role

swbsf opened the pull request:
tinc-dynamic is doing its job pretty well.

Commit:
Removing tinc-static files.

Commit:
Do not call tinc-static role.

Commit:
Removing all tinc-static files

Commit:
idbvpn does no longer needs to be loaded by tinc.

Commit:
Tinc dynamic should now create its own RSA key

letompouce reviewed (dismissed):
Please keep `/etc/logrotate.d/tinc` since we still need it for the `testvpn` and other tinc VPNs!

Commit:
We create a file on the device instead

Commit:
File will be created on the device instead

Commit:
File will be created on the device instead

Commit:
Switch to variable

Commit:
Remove firewall rules

Commit:
Remove old variables

Commit:
Add variable for vpn

Commit:
Get tinc-dynamic more flexible

- Install tincmaster with regular vpn name, port and master server
- Try to loop over a role with with_items but without succes: Need to be
  fix !

Commit:
Move towards more flexibility

Commit:
File with generic name, will be copied with vpn_name

letompouce commented on roles/tinc-dynamic/defaults/main.yml:
```diff
@@ -1,12 +1,7 @@
 # if vpn_log_level is set but vpn_log_file is not,
 # it will use syslog by default.
-vpn_log_file: "/var/log/tinc.{{ vpn_name2 }}.log"
+vpn_log_file: "/var/log/tinc.{{ vpn_name }}.log"
```
Dont provide a path to `--logfile`.

See https://github.com/ideascube/ansiblecube/commit/01e68b8804e234637bffd3acaf82d7d7206bc91f

letompouce commented on roles/set_custom_fact/files/device_list.fact:
```diff
@@ -2960,18 +2960,24 @@
+			"vpn": [{
+				"name": "email",
+				"port": "658",
+				"master_server" : "ruminant"
```
You don't need to define a port and a master for each VPNs.

`email`'s remote port will always be the same, its master will always be `ruminant`. And for the local port, it makes sense to use a defined port per VPN. Like, 655 for `testvpn`, 658 for `email`, and so on.

All of these could defined in a `vars/{{ item }}-vpn.yml` file included using some `with_items: {{  ansible_local.device_list[generic_project_name].vpns }}` (note the plural).

Commit:
Give directly the vpn_name instead

Commit:
Play with blockinfile and content adding

Commit:
Use a file instead to store vpn interface definition

Commit:
Remove loop on the role

Commit:
Remove unecessary notify about firewall closing port

Commit:
Fix the blockinfile marker

Commit:
Remove old vpn interface
